server/src/test/java/org/spine3/server/aggregate/AggregateStorageShould.java

```diff
@@ -253,6 +257,129 @@ public void rewrite_event_count_after_last_snapshot() {
         assertEquals(expectedValue, actualCount);
     }
 
+    @SuppressWarnings("OptionalGetWithoutIsPresent") // Checked in an assertion
+    @Test
+    public void write_entity_status_of_aggregate() {
+        final ProjectId id = Given.newProjectId();
```

Comment:
Why cannot we create the `id` in `setUp()`? We use it almost in every test method.

This would also reduce the need for a specific `Given` class. We have test factory for identifiers. We can call it once in `setUp()` and that would not reduce readability.